project/oooJiun/README.md

```diff
@@ -0,0 +1,76 @@
+# PolyMap: Polygon-Based 2D Map Generator
+
+## Basic Information
+
+- GitHub Repository: https://github.com/oooJiun/PolyMap
+
+PolyMap is a tool that empowers users to create custom 2D maps with a user-friendly Python interface. 
+To achieve this goal, PolyMap will use boolean operations, such as AND, OR, NOT, and XOR, on polygons to generate the map.
```

Comment:
Implementing any of the polygon Boolean operation is not trivial.  It will be a good idea to set individual plan for each of them.

Reply:
ok! I will fix it today